code/app/src/main/res/layout/activity_add_recipe.xml

```diff
@@ -80,38 +81,83 @@
 
     </com.google.android.material.textfield.TextInputLayout>
 
-    <com.google.android.material.textfield.TextInputLayout
-        android:id="@+id/recipe_addIngredients"
+    <LinearLayout
+        android:id="@+id/listview"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:hint="Add Ingredients"
+        android:layout_marginStart="16dp"
         android:layout_marginTop="16dp"
         android:layout_marginEnd="16dp"
-        android:layout_marginStart="16dp"
-        app:layout_constraintTop_toBottomOf="@+id/recipe_category">
+        android:orientation="vertical"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/recipe_category">
 
```

Comment:
Inconsistent background

Reply:
I moved things around to make the button stay where it is rn, but yeah might need a scroll view if instructions are long